src/register.cpp

```diff
@@ -203,6 +203,15 @@ bool TRegister::IsExcludedFromPolling() const
 void TRegister::ExcludeFromPolling()
 {
     ExcludedFromPolling = true;
+    if (SporadicMode == TRegisterConfig::TSporadicMode::ONLY_EVENTS) {
+        LOG(Info) << ToString() << " excluded from polling: register is in sporadic mode";
```

Comment:
```suggestion
        LOG(Debug) << ToString() << " excluded from polling: register is in sporadic mode";
```
Мы уже и так пишем, что включили опрос по событиям